examples/application/community_crime.py

```diff
@@ -0,0 +1,87 @@
+"""
+==============================
+Communities and Crime - sklearn and pyglmnet
+==============================
```

Comment:
maybe nitpick but it would be better visually if the `=` goes till the end of `pyglmnet.